### DISCUSSION TITLE
Multi touch zoom

### DISCUSSION THREAD

**Discussion by mgnslndh** [PanAndZoom/PanAndZoom#67] [Q&A] [answered]
As far as my testing goes I can not get multi touch zoom (pinching) to work with the `ZoomBorder` control. Is this correct or should it work out of the box?

- Is there any plans on adding support for multi touch zoom?
- If you would add support for multi touch zoom would it require some pinch gesture support in Avalonia or could it be supported/implemented independently of Avalonia?

I think at some point this repo had support for touch zoom but it could be a long time ago when there was a WPF version of the control.

**Comment by mgnslndh** [comment]
@wieslawsoltes have you considered adding support for the new `PinchGestureRecognizer`? 

https://github.com/AvaloniaUI/Avalonia/blob/111a03abe89b8c441f2e3ebaa331bc4743cd1010/src/Avalonia.Base/Input/GestureRecognizers/PinchGestureRecognizer.cs

**Comment by wieslawsoltes** [comment, marked as answer]
Should be fixed in latest release
